src/locale/locales/fr/messages.po

```diff
@@ -2350,13 +2365,9 @@ msgstr "Bienvenue sur <0>Bluesky</0>"
 msgid "What is the issue with this {collectionName}?"
 msgstr "Quel est le problème avec cette {collectionName} ?"
 
-#: src/view/com/auth/SplashScreen.tsx:34
-#~ msgid "What's next?"
-#~ msgstr "Et après ?"
-
 #: src/view/com/auth/SplashScreen.tsx:34
 msgid "What's up?"
-msgstr ""
+msgstr "Qu'est-ce qui se passe ?"
```

Comment:
This translates to `What is happening?`. If you want something that is really translating to `What's up?`, I would suggest translating it to `Quoi de neuf ?` instead (literally `What's new?`, but used in the same place as English's `What's up?`.).